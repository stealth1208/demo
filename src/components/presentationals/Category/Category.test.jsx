import React from 'react';
import renderer from 'react-test-renderer';
import { Category } from '../index';

it('renders correctly', () => {
  const categories = [{
    href: "/all-stores",
    iconUrl: "https://cloud.shopback.com/raw/upload/static/images/icon/core/fire.svg",
    id: 0,
    name: "Popular"
  }];
    const tree = renderer
      .create(
        <Category 
          categories={categories}
          selectCategory={() => {console.log('selected')}}
          selectedCategory={0}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });