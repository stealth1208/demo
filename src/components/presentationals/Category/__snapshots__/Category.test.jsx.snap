// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="categories"
>
  <div
    className="category__item category__item--selected"
    onClick={[Function]}
  >
    <div
      className="category__item__image"
    >
      <img
        alt="Popular"
        src="https://cloud.shopback.com/raw/upload/static/images/icon/core/fire.svg"
      />
    </div>
    <a
      className="category__item__link"
    >
      Popular
    </a>
  </div>
</div>
`;
